modules/contact/include/convergence/AugmentedLagrangianContactConvergence.h

```diff
@@ -0,0 +1,43 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ReferenceResidualProblem.h"
+#include "FEProblem.h"
+#include "NodeFaceConstraint.h"
+#include "MechanicalContactConstraint.h"
+#include "AugmentedLagrangianContactProblemInterface.h"
+#include "ReferenceResidualConvergence.h"
+#include "ResidualConvergence.h"
+
+/**
+ * Class to check convergence for the augmented Lagrangian contact problem.
+ */
+template <class T>
+class AugmentedLagrangianContactConvergence : public T,
```

Comment:
Reminder to create a markdown file for this one. Not an error in the CI output, but got flagged as a warning, so I thought I'll mention it along with the other doco fixups 😄 

The markdown file should be placed at: 
```
modules/contact/doc/content/source/convergence/AugmentedLagrangianContactConvergence.md
```

Reply:
added